src/log/Log.h

```diff
@@ -70,6 +72,7 @@ class Log : private Thread
   void set_log_file(std::string fn);
   void reopen_log_file();
   void chown_log_file(uid_t uid, gid_t gid);
+  void set_log_stderr_prefix(const std::string& p);
```

Comment:
the function signature in header file does not match that in .cc file. see https://github.com/ceph/ceph/pull/19385/files#diff-8bc11b017f7a3710bf6737063a301577R136